pkg/cephmgr/osd/scheme.go

```diff
@@ -26,21 +26,29 @@ import (
 )
 
 const (
-	WalPartitionName      = "wal"
-	DatabasePartitionName = "db"
-	BlockPartitionName    = "block"
-	walPartition          = 1
-	databasePartition     = 2
-	blockPartition        = 3
-	UseRemainingSpace     = -1
-	schemeFilename        = "partition-scheme"
-	WalDefaultSizeMB      = 576
-	DBDefaultSizeMB       = 20480
+	Filestore                     = "filestore"
```

Comment:
note @kokhang that I did change the partition names to be enums since there is no user interaction with those types: https://github.com/rook/rook/pull/529/commits/c8276a09854e037eff3372d4da1157de9b3867ac#diff-c981038d0c867b4e1a90b7000123e519R42

I kept the store types (bluestore/filestore) as strings since the user is expected to type them into the cluster TPR yaml.